src/software/field_tests/movement_robot_field_test.py

```diff
@@ -134,7 +134,7 @@ def test_basic_rotation(field_test_runner):
         # validate by eye
         logger.info(f"robot set to {angle} orientation")
 
-        time.sleep(2)
+        # time.sleep(2)
```

Comment:
why is the sleep commented out?

Reply:
Lets just remove it. This was making the test very slow